source/common/stats/allocator_impl.cc

```diff
@@ -240,9 +240,10 @@ class TextReadoutImpl : public StatsSharedImpl<TextReadout> {
   }
 
   // Stats::TextReadout
-  void set(std::string&& value) override {
+  void set(const std::string& value) override {
```

Comment:
let's use absl::string_view for the formal here, since we are just going to copy it anyway.

Reply:
Done